tasks/build-preview-urls-comment.js

```diff
@@ -82,11 +79,8 @@ export const buildPreviewURLComment = (ref) => {
     let comment = `## Branch preview
 
 - [Documentation Site](https://${branchSlug}--spectrum-web-components.netlify.app/)
-- [Storybook](https://${branchSlug}--spectrum-web-components.netlify.app/storybook/)`;
+- [Storybook](https://${branchSlug}--spectrum-web-components.netlify.app/storybook/)
 
-    // If there are changed packages, add a section with visual regression test results
```

Comment:
I believe we should always have links for VRTs posted. For example, some dependency changes don’t trigger `getChangedPackages()`, but they can still cause regressions. Since the VRTs run regardless, it’s just a matter of ensuring we see the links.